README.md

```diff
@@ -1,8 +1,64 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nasr Syed . Jarvis Consulting
 
+My name is Nasr Syed, and I am a highly ambitious Data Engineer looking to break his way into the field of Data Analytics. My academic background consists of a Bachelor of Engineering (B.Eng) in Electrical Engineering from Ryerson University, along with a Data Engineering certification from IBM. In terms of my professional work experience, I have been an Electrical Engineer for about 3 years, with the last 6 months in Project Management at top name firms located in Toronto. As displayed by my successful track record, I am someone who thrives, and pushes himself to be an engineer of the highest caliber. Now having found true passion, I am determined to make an impact in the world of big data.
```

Comment:
My name is Nasr Syed, and I am a highly ambitious Software Engineer looking to break his way into the field of Data Analytics. My academic background consists of a Bachelor of Engineering (B.Eng) in Electrical Engineering from Ryerson University and a Data Engineering certification from IBM. In terms of my professional work experience, I have been an Electrical Engineer for about three years, with the last six months in Project Management at top-name firms located in Toronto. As displayed by my successful track record, I am someone who thrives and pushes himself to be an engineer of the highest calibre. Now having found a true passion, I am determined to impact the world of big data.
 

minor fix.